docs/src/examples/components/Dropdown/Types/DropdownExampleMultipleSelection.shorthand.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { Dropdown } from '@stardust-ui/react'
+
+const inputItems = [
+  'Bruce Wayne',
+  'Natasha Romanoff',
+  'Steven Strange',
+  'Alfred Pennyworth',
+  `Scarlett O'Hara`,
+  'Imperator Furiosa',
+  'Bruce Banner',
+  'Peter Parker',
+  'Selina Kyle',
+]
+
+const DropdownExample = () => (
+  <Dropdown
+    multiple
+    items={inputItems}
+    placeholder="Select your heroes"
+    getA11ySelectionMessage={getA11ySelectionMessage}
+    noResultsMessage="We couldn't find any matches."
+  />
+)
+
+const getA11ySelectionMessage = {
+  onAdd: item => `${item} has been selected.`,
+  onRemove: item => `${item} has been removed.`,
+}
```

Comment:
Do we need `getA11ySelectionMessage()` for _this example_?

Reply:
removed